index.html

```diff
@@ -114,7 +114,63 @@ <h3>This is a track title</h3>
     <br><br>
     <p class="test-text">Example text for DINPro mixin</p>
     <br>
-    <div class="circle"></div>
+
+    <div class="intro">
+      <div class="container" style="background-color: white;">
+
+          <div class="row">
+            <div class="col-sm" style="padding-right: 35px;">
+              <div class="intro-header" style="padding-bottom: 1%">
+                <h1>What is Technica?</h1>
+              </div>
+              <p>Technica is the world’s largest all-women and nonbinary hackathon, hosted annually at the University of Maryland.</p>
+              <br>
+              <p>Over the duration of 24 hours, women and non-binary people are immersed in tech culture and encouraged to exercise their imagination to create interesting and innovative hacks.</p>
+              <br>
+              <p>This year’s theme is “Expand Your Horizons”. At Technica, we want to see women and non-binary folks thrive and go beyond society’s expectations for them. We encourage all attendees to take your tech journey beyond Technica. Don’t stop at our hackathon, attend others and keep creating. Technica is meant to catapult you further in your career, and help you to continue achieving. As you advance in your career, go beyond your duties at work, become a and help to advance the next generation of women in tech. Whatever your goals and aspirations, remember you can always #Expand #Your Horizon</p>
+            </div>
+            <div class="col-sm" style="padding-top: 64px;">
+              <div class="rectangle"></div>
```

Comment:
oops heh so in the wireframe this grey square was a placeholder for the youtube video from last year. https://www.youtube.com/watch?v=Rcoc0Yo37Nk can you stick the code to embed this here? (can find the code by pressing the share button -> pressing embeded)